packages/nimble-components/src/wafer-map/tests/wafer-map.spec.ts

```diff
@@ -85,6 +87,41 @@ describe('WaferMap', () => {
             expect(spy).toHaveBeenCalledTimes(1);
         });
 
+        it('will have list render strategy after dies change', () => {
+            element.dies = [{ x: 1, y: 1, value: '1' }];
+            processUpdates();
+            expect(element.renderStrategy).toEqual('list');
+        });
+
+        it('will update once after diesTable change', () => {
+            element.diesTable = new Table();
+            processUpdates();
+            expect(spy).toHaveBeenCalledTimes(1);
+        });
+
+        it('will have matrix render strategy after diesTable change', () => {
+            element.diesTable = new Table();
+            processUpdates();
+            expect(element.renderStrategy).toEqual('matrix');
+        });
+
+        xit('will trigger render-complete after diesTable change', async () => {
```

Comment:
@atmgrifter00 this test is failing waiting on the promise. I implemented this the same way that the toggle event was tested. Is there something else I should do?

Reply:
I'll have to pull down your branch and play with it (which I haven't been able to do yet).

Reply:
I removed the event and made a spy instead